tasks/instagrampoll.php

```diff
@@ -60,134 +60,137 @@ public function cleanup()
-							{
-								//Get Model Of Tag if it already exists
-								$tag_id = $check_tag_existance->id;
-								$tag_model = \Propeller\Instagram\Model_Tag::find($tag_id);
-							}
-							else
-							{
-								//Create That Tag if it doesnt already exist
-								$tag_model = \Propeller\Instagram\Model_Tag::forge();
-								$tag_model->tag_name = $tag;
-								$tag_model->save();
-							}
+							$image = \Propeller\Instagram\Model_Image::forge();
+							$image->thumb_img = $med->images->thumbnail->url;
+							$image->main_img = $med->images->standard_resolution->url;
+							$image->lowres_img = $med->images->low_resolution->url;
+							$image->instagram_id = $med->id;
+							$image->author = $med->user->username;
+							$image->link = $med->link;
+							$image->accepted = 'unsorted';
+							$image->subscription_id = $sub->instagram_subscription_id;
+							$image->caption = $med->caption->text;
 
-							//Save Tags 
-							$image->tags[] = $tag_model;
 							$image->save();
-						}
 
-						$count++;
-					}				
+							//Loop Thorugh Tags and store each one
+							foreach($med->tags as $tag) 
+							{
+								//Check Tag Existance
+								$check_tag_existance = \Propeller\Instagram\Model_Tag::query()->where('tag_name', $tag)->get_one();
+								if ($check_tag_existance) 
+								{
+									//Get Model Of Tag if it already exists
+									$tag_id = $check_tag_existance->id;
+									$tag_model = \Propeller\Instagram\Model_Tag::find($tag_id);
+								}
+								else
+								{
+									//Create That Tag if it doesnt already exist
+									$tag_model = \Propeller\Instagram\Model_Tag::forge();
+									$tag_model->tag_name = $tag;
+									$tag_model->save();
+								}
+
+								//Save Tags 
+								$image->tags[] = $tag_model;
+								$image->save();
+							}
+
+							$count++;
+						}							
 				}
-				catch (ImageAccessDeindedException $e)
+
+				if($count) 
 				{
-					\Cli::write("Attempting to delete image", "cyan");
+					$sub->last_image_received = time();
+					$sub->save();
+				}
+
+				\Cli::write($count ? $count.' updates for tag: '.$sub->object_id : 'No updates for tag: '.$sub->object_id, 'green');
+
+			}
+		}
+		catch (ImageAccessDeniedException $e)
+		{
+			\Cli::write("Attempting to delete image", "cyan");
 					
-					if($image) 
-					{
-						$obj = \Propeller\Instagram\Model_Image::find($image->id);
+			if($image) 
+			{
+				$obj = \Propeller\Instagram\Model_Image::find($image->id);
 						
-						// Delete the image as we no longer have access to it!
-						try
-						{
-							$obj->delete();				
+				// Delete the image as we no longer have access to it!
+				try
+				{
+					$obj->delete();				
 							
-							\Cli::write("Image deleted", "green");
-						}
-						catch(\Exception $e)
-						{
-							\Cli::error("Unable to delete image: " . $e->getMessage());
-						}
-					}				
+					\Cli::write("Image deleted", "green");
 				}
 				catch(\Exception $e)
 				{
```

Comment:
I know this bit didn't change, but could we keep all braces for anything other than functions and classes on the same line please?